cylc/flow/network/schema.py

```diff
@@ -2224,7 +2229,7 @@ def delta_subs(root, info: 'ResolveInfo', **args) -> AsyncGenerator[Any, None]:
 
 class Pruned(ObjectType):
     class Meta:
-        description = """WFS Nodes/Edges that have been removed."""
+        description = 'WFS Nodes/Edges that have been removed.'
```

Comment:
What's "WFS" ?  Workflow Service?   Should be "scheduler"? 

```
> git grep WFS
cylc/flow/network/schema.py:        description = """WFS Nodes/Edges that have been removed."""
cylc/flow/network/schema.py:        description = """Grouped deltas of the WFS publish"""
```

Reply:
Absolutely no idea!

Reply:
```suggestion
        description = sstrip('''
            Objects (e.g. workflows, tasks, jobs) which have been removed from
            the store.       
        ''')   
```

Reply:
Ping @dwsutherland - I think you may have been the original author?

Reply:
This is the description for pruned deltas.

These list Cylc "objects" that have drifted out of the window.

I expect WFS probably meant Workflows?

Reply:
I have feeling it date backs to early Cylc 8 planning days when we were considering various other names for what is now just the "scheduler" (Workflow Service = WFS?).

Reply:
Yeah.. Can't recall 😆 

But "Workflow's" might have been it ...

Reply:
Ok screw it, merge Oliver's tweak.